app/jobs/populate_issues_job.rb

```diff
@@ -1,9 +1,5 @@
 class PopulateIssuesJob < ApplicationJob
   def perform(repo)
-    begin
-      repo.populate_multi_issues!(state: 'open')
-    rescue GitHubBub::RequestError => e
-      repo.update_attributes(github_error_msg: e.message)
-    end
+    PopulateIssues.(repo)
```

Comment:
This is actually one of my favorite little ruby-isms: `Foo.()` will call `Foo.call`. I'd be curious to hear if @schneems uses it. When a class has a single entry point that would otherwise be called `call` or `run`, I often prefer this approach because of how nicely it reads.